tests/includes/PdoDatabaseTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/******************************************************************************
+ * Wikipedia Account Creation Assistance tool                                 *
+ *                                                                            *
+ * All code in this file is released into the public domain by the ACC        *
+ * Development Team. Please see team.json for a list of contributors.         *
+ ******************************************************************************/
+
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 12/24/16
+ * Time: 00:52
+ */
```

Comment:
This header is unnecessary.

If you want to disable it, just empty the PHP File Header template:
![image](https://cloud.githubusercontent.com/assets/722710/23096490/ae3d2a6e-f615-11e6-94ac-edaa4b0296cf.png)